web/src/main/ui/app/components/flows-new/models/step.model.ts

```diff
@@ -11,5 +16,29 @@ export class Step {
   public isValid: boolean = false;
   public version: string;
 
-  public options: Options;
-}
\ No newline at end of file
+  public options: any = {};
```

Comment:
This is not good design.

I suggest creating a static factory method to build Step object based on { id, type, name, ...} and derive options based on the provided type property. Make constructor private too to prevent direct construction.